src/axom/sidre/core/Group.hpp

```diff
@@ -1358,6 +1398,21 @@ class Group
    */
   bool createNativeLayout(Node& n, const Attribute* attr = nullptr) const;
 
+  /*!
+   * \brief Deep-copy Group's native layout to given Conduit node.
+   *
+   * This is similar to createNativeLayout, except for the leaves being
+   * deep-copied.
+   *
+   * The destination's allocator id should be preserved and used for
+   * array allocations.
+   *
+   * \return True if the Group or any of its children were added to the Node,
+   * false otherwise.
+   *
+   */
+  bool deepCopyToConduit(Node& dst, const Attribute* attr = nullptr) const;
```

Comment:
Please update the sphinx docs with all the various ways to deep/shallow copy between sidre and conduit.

Reply:
This should be a part of a greater task to rethink sidre/conduit hierarchy copies.  I created an issue for this, https://github.com/LLNL/axom/issues/1633